charts/yeti/templates/nginx-configmap.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "yeti.fullname" . }}-nginx-configmap
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "yeti.labels" . | nindent 4 }}
+data:
+  default.conf: |
+    server {
+
+        root /www;
+
+        location /api/v2 {
+            proxy_pass http://{{ include "yeti.fullname" . }}-api:{{ include "yeti.service.port" . }};
+        }
+
+        location / {
+            try_files $uri $uri/ /index.html;
+        }
+    }
```

There's been a small update to this file! https://github.com/yeti-platform/yeti-docker/blob/main/prod/frontend/nginx.conf

```suggestion
data:
  default.conf: |
    server {

        root /www;

        location /api/v2 {
            proxy_pass http://{{ include "yeti.fullname" . }}-api:{{ include "yeti.service.port" . }};
        }
        
        location ~(^/docs|^/openapi.json) {
            proxy_pass http://api:8000;
        }

        location / {
            try_files $uri $uri/ /index.html;
        }
    }
```